ucp/_libs/ucp_py.pyx

```diff
@@ -44,6 +49,10 @@ UCP_INITIALIZED = False
 LOGGER = None
 listener_futures = set()
 
+
+def get_ucp_worker():
+    return <size_t>get_worker()
```

Comment:
Do we need to explicitly cast this as `<size_t>` ?

Reply:
For this case, since `get_worker()` returns a C++ pointer and we are returning from a pure Python function, the Cython compiler complains unless we cast it to something that can be implicitly converted to a native Python type. I am still planning on looking into John's proposed solution to this.